CLAUDE.md

```
@@ -1,185 +1,51 @@
 # CLAUDE.md
 
-This file provides guidance to Claude Code (claude.ai/code) when working with code in this repository.
-
-## Project Overview
-
-VegaFusion is a Rust-based library that provides Python, JavaScript, and Rust APIs for analyzing and scaling Vega visualizations. The project optimizes Vega specifications by pre-computing data transformations and pushing computations to more efficient backends like Apache DataFusion.
+VegaFusion optimizes Vega visualizations by pre-computing data transforms using Apache DataFusion. This enables scalable charting with the Vega ecosystem.
 
 ## Architecture
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)